string music_number
float32 volume
float32 speed
---
bool success_flag
